blog/2024-09-24-climate-set-temp-validation.md

```diff
@@ -0,0 +1,14 @@
+---
+author: G Johansson
+authorURL: https://github.com/gjohansson-ST
+authorImageURL: https://avatars.githubusercontent.com/u/62932417?v=4
+authorTwitter: GJohansson
+title: "Additional validation in Climate set temperature"
+---
+
+As of Home Assistant Core 2024.10, we have implemented further validation into the `set_temperature` method. This means that the integrations does not need to implement this validation in their own methods.
+
+Currently it was possible to use the `temperature` attribute when the entity did not set `ClimateEntityFeature.TARGET_TEMPERATURE` or `target_temp_low`/`target_temp_high` when the entity did not set `ClimateEntityFeature.TARGET_TEMPERATURE_RANGE`. This has been deprecated and will raise an error starting from 2025.4.
```

Comment:
Implemented also in 2025.10 so I think it was fitting to include that also on this blog post